README.md

```diff
@@ -11,15 +11,31 @@ Tested with Python 3.11.4
 The following steps are not required but recommended. This will allow you to install packages in your isolated virtual environment instead of globally, reducing the risk of breaking system tools or other projects.
 
 1. Install [pyenv](https://github.com/pyenv/pyenv) and the [pyenv-virtualenv](https://github.com/pyenv/pyenv-virtualenv) plugin.
-2. Download the appropriate Python version with `pyenv install 3.11.4` in the command line.
-3. Create a virtual environment with the appropriate Python version and name for your environment, for example `pyenv virtualenv 3.11.4 innovationserhebung`
-4. Activate the environment with `pyenv activate innovationserhebung`
+2. Download the appropriate Python version with 
+```shell
+pyenv install 3.11.4
+``` 
+in the command line.
+3. Create a virtual environment with the appropriate Python version and name for your environment, for example 
+```shell
+pyenv virtualenv 3.11.4 innovationserhebung
+```
+4. Activate the environment with 
+```shell
+pyenv activate innovationserhebung
+```
 
 ### Run the APP
 
 1. Move to the `/app` directory.
-2. Install the required libraries with the command line `pip install -r requirements.txt`
-3. Run the server with the command line `uvicorn main:app --reload`
+2. Install the required libraries with the command line 
+```shell
+pip install -r requirements.txt
```

Comment:
This command needs to be run in the root directory of the repo and not inside `/app` (as indicated in step 1). So, maybe the steps could be adjusted to something like this?

1. Move to the root directory of this repository.
2. Install the required libraries with the command line 
    ```shell
    pip install -r requirements.txt
    ```
3. Move to the `/app` directory.
4. Run the server with the command line

Reply:
done!